pkg/log/config_test.go

```diff
@@ -220,6 +221,33 @@ func TestRotateAndStdout(t *testing.T) {
 	}
 }
 
+func TestGlogV(t *testing.T) {
+	o := DefaultOptions()
+	_ = Configure(o)
+
+	for i := 0; i < 10; i++ {
```

Comment:
this is probably a bit of overkill, based on our implementation (as i understand it), but seems fine to keep in case that changes.